src/assets/src/components/queueSettings.tsx

```diff
@@ -144,6 +146,7 @@ export function ManageQueueSettingsPage(props: PageProps<SettingsPageParams>) {
                 setName(q.name);
                 setDescription(q.description);
                 setAllowedMeetingTypes(new Set(q.allowed_backends));
+                setPhysLocation(q.physLocation);
```

Comment:
```suggestion
                setPhysLocation(q.phys_location);
```
To work with other changes.